src/refactor_nrepl/config.clj

```diff
@@ -34,7 +34,14 @@
    :libspec-whitelist ["^cljsjs"]
 
    ;; Regexes matching paths that are to be ignored
-   :ignore-paths []})
+   :ignore-paths []
+
+   ;; Will be forwarded to clojure.pprint/*print-miser-width* when pprinting ns forms.
+   ;; You can set it to 40 for restoring traditional behavior.
+   :print-miser-width nil
```

Comment:
I guess this should also be documented in the README.

Reply:
Should probably mention this default change in the changelog too, like you did for `print-right-margin`